Dockerfile

```diff
@@ -32,17 +22,28 @@ RUN uv sync --frozen --no-group dev --no-install-project --python /usr/bin/pytho
 # the source code
 COPY README.md ./
 COPY er_aws_kms ./er_aws_kms
-COPY module ./module
 
 # Sync the project
 RUN uv sync --frozen --no-group dev
 
+# Copy the module directory and set 777 permissions.
+USER 0
+COPY module ./module
+RUN chmod 777 ./module
```

Comment:
Please use `COPY --chmod` (https://docs.docker.com/reference/dockerfile/#copy---chown---chmod) instead of `RUN chmod`.

Reply:
Added.